core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -198,3 +199,116 @@ describe('Admin backend api service', () => {
     })
   );
 });
+
+describe('Admin Backend API service for Jobs Tab', () => {
+  let adminBackendApiService: AdminBackendApiService;
+  let httpTestingController: HttpTestingController;
+  let csrfService: CsrfTokenService = null;
+  let successHandler = null;
+  let failHandler = null;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [AdminBackendApiService]
+    });
+    httpTestingController = TestBed.get(HttpTestingController);
+    adminBackendApiService = TestBed.get(AdminBackendApiService);
+    csrfService = TestBed.get(CsrfTokenService);
+    successHandler = jasmine.createSpy('success');
+    failHandler = jasmine.createSpy('fail');
+
+    spyOn(csrfService, 'getTokenAsync').and.callFake(() => {
+      return Promise.resolve('sample-csrf-token');
+    });
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+  it('should start the new job',
+    fakeAsync(() => {
+      let jobType = 'ActivityContributorsSummaryOneOffJob';
+      adminBackendApiService.startNewJobAsync(jobType)
+        .then(successHandler, failHandler);
+
+      let req = httpTestingController.expectOne(
+        '/adminhandler');
```

Comment:
```suggestion
      let req = httpTestingController.expectOne('/adminhandler');
```

Try to use the inline space wherever possible. (Ditto elsewhere)

Reply:
done!